docs/2024/spdx-expression/updates/2024-06-25.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Week 5
+author: Akash Sah
+tags: [gsoc24, spdx-expression]
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2024 Akash Sah <akashsah2003@gmail.com>
+-->
+
+# Week 5 Meeting and Activities
+
+_(June 25, 2024)_
+
+## Attendees:
+
+- [Akash Sah](https://github.com/Akashsah2003)
+- [Gaurav Mishra](https://github.com/GMishx)
+- [Shaheem Azmal M MD](https://github.com/shaheemazmalmmd)
+- [Dearsh Oberoi](https://github.com/deo002)
+- [Abhishek Kumar](https://github.com/abhi-kumar17871)
+
+## Discussions
+
+### Presentation of Report Generation
+
+- Presented my work on SPDX Tag Value and CSV reports, including license expression results in the reports.
+- Feedback: There should be functionality for adding license expressions and main licenses.
+- I have validated SPDX Tag Value reports using [SPDX online tools](https://tools.spdx.org/app/).
+- Inquired about how to validate CSV formats.
+- Next steps: Work on different formats.
+
+## Activities
+
+- Opened a WIP PR [feat(spdx-expression): License Expression Support](https://github.com/fossology/fossology/pull/2771).
+- Solved some bugs found during the generation of SPDX Tag Value reports.
+- Worked on SPDX RDF reports and successfully included license expression results in the reports and validated the reports.
+- Added license exception support in the reports.
+- Added an option to make a license expression the main license in the UI.
+
+## Sample Reports
+
+- [SPDX Tag Value Sample](https://drive.google.com/file/d/1Trnamycdj9MEAz4Um6XN7wb3JxP2tSNf/view?usp=sharing)
+- [SPDX RDF Sample](https://drive.google.com/file/d/1eqS51s66ypKN-k9VioqjKJClImb8q5AF/view?usp=sharing)
```

Comment:
If the files are not very big, will recommend committing it to repo itself. It will help preserve it for long.

Reply:
Added sample reports in static directory.